sw/banshee/src/tran.rs

```diff
@@ -1215,6 +1215,7 @@ impl<'a> InstructionTranslator<'a> {
         match data.op {
             riscv::OpcodeImm12Rs1::Scfgwi => {
                 // ssr write immediate holds address offset in imm12, content in rs1
+                // rs2[11:5]=reg_word rs2[4:0]=dm -> addr_off = {dm, reg_word[4:0], 000}
```

Comment:
Ah and sorry Noah for being extremely nitpicky, but could you change the comment as follows here and in the read function as well?

```suggestion
                // imm12[11:5]=reg_word imm12[4:0]=dm -> addr_off = {dm, reg_word[4:0], 000}
```

Reply:
:+1: 911c1b9